README.md

```diff
@@ -200,6 +200,47 @@ You can use the following concrete class:
     ProcessBuilder process = new ProcessBuilder("java","-jar","path/to/language-server.jar");
     new ProcessBuilderServerDefinition("bsl,os", process);
     ```
+
+- **Custom Initialization Options**
+
+    If your LSP server needs some custom initialization options when connecting, 
+    you can define a class that extends `ProcessBuilderServerDefinition` or `RawCommandServerDefinition`, 
+    and then override the `getInitializationOptions` method. 
+    Here's an example:
+
+    ```java
+    public class MyServerDefinition extends ProcessBuilderServerDefinition {
+        
+        /**
+         * Creates a new instance.
+         *
+         * @param ext     The extension.
+         * @param process The process builder instance to be started.
+         */
+        public MyServerDefinition(String ext, ProcessBuilder process) {
+            super(ext, process);
+        }
+        
+        /**
+         * Returns the custom initialization options for the given URI.
+         *
+         * @param uri The URI.
+         * @return The initialization options.
+         */
+        @Override
+        public Object getInitializationOptions(URI uri) {
+            // Return your custom initialization options here.
```

Comment:
I think this should provide an actual example. For example: how would someone set the `InitializeParams#clientInfo` field's value?